ncm-metaconfig/src/main/metaconfig/chrony/pan/schema.pan

```diff
@@ -0,0 +1,85 @@
+declaration template metaconfig/chrony/schema;
+
+include 'pan/types';
+
+type chrony_service_server_flag = choice(
+    'iburst', 'burst', 'offline', 'auto_offline', 'prefer', 'noselect', 'trust', 'require', 'xleave');
+
+@documentation{
+    the crony.conf configuration entry for server/pool/peer options
+}
+type chrony_service_server_options = {
+    'minpoll' ? long = -6 # power of 2 (1/64th of a second)
```

Comment:
If these are not mandatory then would it be better to leave them without defaults and thus allow the upstream default to take precedence? I think we've had issues in the past where we've set a default for a piece of software, only for a later version of RHEL/SL to set a different default which we've then not picked up.